extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/ranger/AccessRequest.scala

```diff
@@ -81,23 +74,11 @@ object AccessRequest {
 
   private def getUserGroupsFromUserStore(user: UserGroupInformation): Option[util.Set[String]] = {
```

Comment:
here we use util.Set, how about bringing into correspondence with `java.util.{Set => JSet}`

Reply:
addressed.